lib/zstd.h

```diff
@@ -408,10 +408,12 @@ ZSTDLIB_API size_t ZSTD_DStreamOutSize(void);   /*!< recommended size for output
 #define ZSTD_FRAMEHEADERSIZE_PREFIX 5   /* minimum input size to know frame header size */
 #define ZSTD_FRAMEHEADERSIZE_MIN    6
 #define ZSTD_FRAMEHEADERSIZE_MAX   18   /* for static allocation */
-static const size_t ZSTD_frameHeaderSize_prefix = ZSTD_FRAMEHEADERSIZE_PREFIX;
-static const size_t ZSTD_frameHeaderSize_min = ZSTD_FRAMEHEADERSIZE_MIN;
-static const size_t ZSTD_frameHeaderSize_max = ZSTD_FRAMEHEADERSIZE_MAX;
-static const size_t ZSTD_skippableHeaderSize = 8;  /* magic number + skippable frame length */
+
+#define ZSTD_frameHeaderSize_prefix ZSTD_FRAMEHEADERSIZE_PREFIX
```

Comment:
Do we want to keep these around?

Reply:
That's the intention : any code using `ZSTD_frameHeaderSize_prefix` will continue to work properly with this `#define`.

Now, since it's in the `ZSTD_STATIC_LINKING_ONLY` section, we have no obligation.
It's just that I felt it would be "nicer" for any potential user of these constants.
I have no strong feeling about it.